### ISSUE TITLE
Final Report Peer Review - Zongyuan Yuan (zy225)

### ISSUE THREAD

**Issue by benyuan98** [yujin465/wineanalysis#15]
This is a very interesting project and the report is very well written, in terms of both formatting and the content. The descriptions of the dataset and EDA were illustrative. It was thoughtful of the team to speculate that the relationship of features and wine-quality is non-linear and then applied polynomial transformation to the original dataset. Not only did they attempt various models including linear models, tree-based models, KNN and GaussianSVM that achieved good performance on the test set, they have also done a great job at explaining how they tuned the hyperparameters and avoided overfitting. Most remarkably, they have conducted production optimization that were able to recommend the optimal values of the features to maximize the scores indicating the quality of wine. Therefore, the results of this project are instructive and of high practical values for both customers and wine producers. Specifically, customers can better understand the quality of wine and wine producers can know how to make better wine. Overall, this is an excellent project and I really enjoy it.
